backend/offchain/types/__init__.py

```diff
@@ -161,6 +164,11 @@ def _from_dict(
         )
     return klass(**obj)
 
+COMMAND_RESPONSE_RESULTS = {
```

Comment:
all our types definitions are located at backend\offchain\types\__init__.py, I think we should move this definitions to there and keep the naming conventions (_RESULT_TYPES like we have _OBJECT_TYPES and _RESULT_TYPE_FIELD_NAME like we have _OBJECT_TYPE_FIELD_NAME or something similar)

Reply:
Thanks for the review!

Regarding the location - not sure I understand - my change is in `backend/offchain/types/__init__.py`...

Regarding the naming - fixed!